src/Catty/Parser&Serialializer/New Parser & Serializer/XMLHandler/Scene/Scene+CBXMLHandler.m

```diff
@@ -0,0 +1,156 @@
+/**
+ *  Copyright (C) 2010-2020 The Catrobat Team
+ *  (http://developer.catrobat.org/credits)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+
+#import "Scene+CBXMLHandler.h"
+#import "GDataXMLElement+CustomExtensions.h"
+#import "CBXMLValidator.h"
+#import "CBXMLParserHelper.h"
+#import "CBXMLParserContext.h"
+#import "CBXMLSerializerContext.h"
+#import "CBXMLPositionStack.h"
+#import "CBXMLSerializerHelper.h"
+#import "SpriteObject.h"
+#import "UserDataContainer+CBXMLHandler.h"
+#import "Pocket_Code-Swift.h"
+
+@implementation Scene (CBXMLHandler)
+
+#pragma mark - Parsing
++ (instancetype)parseFromElement:(GDataXMLElement*)xmlElement withContext:(CBXMLParserContext*)context
+{
+    Scene *scene = nil;
+    if ([xmlElement.name  isEqual: @"scene"]){
```

Comment:
Is this already a preparation for the new XML format?

Reply:
Yes